autoendpoint/src/extractors/notification.rs

```diff
@@ -1,6 +1,6 @@
 use crate::error::{ApiError, ApiErrorKind};
-use crate::server::extractors::notification_headers::NotificationHeaders;
-use crate::server::extractors::subscription::Subscription;
+use crate::extractors::notification_headers::NotificationHeaders;
+use crate::extractors::subscription::Subscription;
```

Comment:
optional, could probably collapse these:
```
use crate::extractors::{
    notification_headers:::NotificationHeaders,
    subscription::Subscription,
};
```
if you do, just need to be consistent with other, similar includes. Could save a bit of typing for any future moves like this.

Reply:
We can make rustfmt handle this automatically:
https://rust-lang.github.io/rustfmt/?version=master&search=#merge_imports

Reply:
Yeah, it's the clippy battle. I think clippy really wanted folks to collapse when possible, probably because of tons of one line `use`s.

Reply:
Oh, is Clippy complaining about this? I don't see any relevant lints in https://rust-lang.github.io/rust-clippy/stable/index.html